app/assets/javascripts/components/revisions/revision.jsx

```diff
@@ -24,7 +24,7 @@ const Revision = ({ revision, index, wikidataLabel, course, setSelectedIndex, la
       </td>
       <td className="desktop-only-tc">{revision.revisor}</td>
       <td className="desktop-only-tc">{revision.characters}</td>
-      <td className="desktop-only-tc">{revision.references}</td>
+      <td className="desktop-only-tc">{revision.features['feature.wikitext.revision.ref_tags'] || 0 - revision.features_previous['feature.wikitext.revision.ref_tags'] || 0}</td>
```

Comment:
This probably makes sense to do either on the server, at the point of serializing the revisions to json, or if it's to be done client-side, do it in a selector.

I think server-side is preferable, as you can rely on the `Revisions` method that you already wrote.